graylog2-web-interface/src/components/event-definitions/event-definition-form/ShareForm.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React from 'react';
+
+import { Col, Row } from 'components/bootstrap';
+import EntityCreateShareFormGroup from 'components/permissions/EntityCreateShareFormGroup';
+import type { EntitySharePayload } from 'actions/permissions/EntityShareActions';
+
+import commonStyles from '../common/commonStyles.css';
```

Comment:
I am not sure if it's a good idea to keep supporting these commonStyles, I think it's better to create a reusable styled component.

Reply:
Thank you. Refactoring this would involve touching many parts outside the scope of this PR. I will create an issue for it.